RollbarPLCrashReporter/Sources/RollbarPLCrashReporter/RollbarPLCrashCollector.m

```diff
@@ -20,11 +20,17 @@ +(void)initialize {
         
         // Configure our reporter:
         PLCrashReporterSignalHandlerType signalHandlerType =
-#if !TARGET_OS_TV
-            PLCrashReporterSignalHandlerTypeMach;
-#else
-            PLCrashReporterSignalHandlerTypeBSD;
-#endif
+        // NOTE: looking at comments around the PLCRASH_FEATURE_MACH_EXCEPTIONS
+        //       definition in PLCrashReporter and matching it with actual experiences
+        //       on the current Apple OSs releases we should only use
+        //       the PLCrashReporterSignalHandlerTypeBSD option for now.
+        //       Leaving the alternative commented out for the reference in the future.
+//#if PLCRASH_FEATURE_MACH_EXCEPTIONS
```

Comment:
clean up?

Reply:
Nope :), the comments above the commented out code explain why we keep it (at least for now).